src/com/gfredericks/test/chuck/clojure_test/impl.cljc

```diff
@@ -0,0 +1,29 @@
+(ns com.gfredericks.test.chuck.clojure-test.impl
+  (:require #?(:clj  [clojure.test :refer [is]]
+               :cljs [cljs.test :refer-macros [is]])))
+
+;; copied from clojure.test.check, which privatized the function in
+;; recent versions.
+;;
+;; I think there might be plans for test.check to abstract this logic
+;; into a protocol or something, so I'm not too bothered by the
+;; copypasta for now.
+(defn ^:private not-falsey-or-exception?
+  [value]
+  (and value (not (instance? #?(:clj  Throwable
+                                :cljs js/Error)
```

Comment:
and test.check even on clj already has a converse problem where your test can't return (truthy) `Exception` objects without them being confused for thrown exceptions, so this is definitely okay for now.